src/python/scripts/concat_constrained_elems.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Concatenate the constrained elements from each Genomic Align Block (GAB) into one file for each species
+present in the alignment
+
+The coordinates of each constrained element are translated from the GAB coordinate level to the Genome
+coordinate level of each species and then they are concatenated into one file per species
+
+Typical usage example:
+
+    $ python concatenate_constrained_elements.py --json_desc gab_ce.txt --out_dir ./out/
+
+"""
+
+import argparse
+import json
+from typing import Dict
+
+from ensembl.compara.utils import alignment_to_seq_region
+
+
+def translate_ce_coordinates(json_conf: str) -> Dict:
+    """Project each of the constrained elements to the unaligned genomic coordinates in each
+    genome/species aligned
+
+    Args:
+        json_conf: file storing the list of json files to analyse
+
+    Returns:
+        A dictionary of assembly name to constrained element at the genomic coordinate level for the named
+        assembly.The constrained element is a list of the type [seq_name, start, end, score, pval, strand]
+
+    """
```

Comment:
```suggestion
    """Returns a dictionary of assembly name to constrained element at the unaligned genomic coordinate
    level for the named assembly.
    The constrained element is a list of the type [seq_name, start, end, score, pval, strand].

    Args:
        json_conf: file storing the list of json files to analyse
    """
```
Again with the compara convention for docstrings in which the return statement is part of the first line of the docstring.

Reply:
I always put a return statement in my docstring and this is what we have to follow. We do not put a return statement if the function does not return anything.